docs/sidebars.js

```diff
@@ -75,20 +75,6 @@ module.exports = {
             type: 'category',
             label: 'Guides',
             items: [
-                {
```

Comment:
what's this change for? Does it change the URL?

Reply:
It doesn't change the URL. And I've been meaning to do it for a while now - it just feels weird inside of the `guides` submenu (it feels a bit too hidden). 

I can revert it if you don't like it

Reply:
makes sense! I was checking if this was intended because it wasn't related to the PR